src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
> WIP slides are here

Slides look good!

The directive syntax was changed yesterday to ``@{...}`` ([here](https://github.com/MichaelHatherly/Docile.jl/blob/9735205066e082ef30317c9c8534cf33f6e73524/src/Directives/parser.jl#L2)) since I think that hints a more clearly at what is happening, ie. a "macro"-like transformation. Also, that syntax is an error in Julia so it's not likely to come up in docstrings I don't think. Thoughts?

Hopefully the directives that I've written so far, in [extensions.jl](https://github.com/MichaelHatherly/Docile.jl/blob/9735205066e082ef30317c9c8534cf33f6e73524/src/Directives/extensions.jl), illustrate how to add new ones. The ``repl`` and ``example`` ones could still use a little work though.

> assuming I'm helping

Definitely. Testing this stuff out on real docs is what's needed at the moment.